### ISSUE TITLE
Easyjson 0.7.2 emits broken file header

### ISSUE THREAD

**Issue by stefanb** [mailru/easyjson#302]
Easyjson 0.7.2 emits file header with some extra (debug?) information. Such code will NOT match the "generated code" regex, which is used by various code editors to warn editors to not modify this file manually:
https://golang.org/pkg/cmd/go/internal/generate/
RegEx: `^// Code generated .* DO NOT EDIT\.$`

Example:
```go
//v9: false//v28: false//v32: false//v37: false//v45: false//v49: false//v55: false//v58: false//v62: false//v64: false//v74: false//v92: false//v95: false//v100: false//v105: false//out.Ver: false//v110: false//v116: false//v122: false//v126: false//v132: false//v134: false//v138: false//out.H: false//out.W: false//v144: false//v147: false//v153: false//v165: false//v186: false//v205: false// Code generated by easyjson for marshaling/unmarshaling. DO NOT EDIT.
```

With 0.7.1 it was ok:
```go
// Code generated by easyjson for marshaling/unmarshaling. DO NOT EDIT.
```

**Comment by stefanb** [comment]
This seems to be coming from https://github.com/mailru/easyjson/blob/87794d0ae89edce44f4bf90b676a1c080bda5e44/gen/decoder.go#L302

**Comment by GoWebProd** [comment]
Hello. Thanks for your report. I think that #300 fix your issue

**Comment by zikaeroh** [comment]
Yep, this is fixed in 0.7.3.
